course_catalog/models.py

```diff
@@ -74,12 +88,105 @@ class Course(TimestampedModel):
     url = models.URLField(null=True, max_length=2048)
     program_type = models.CharField(max_length=32, null=True, blank=True)
     program_name = models.CharField(max_length=256, null=True, blank=True)
-    location = models.CharField(max_length=128, null=True, blank=True)
```

Comment:
I think we should keep this for now, I'll explain in my overall feedback.

Reply:
I had the same thought initially, but after taking a look at `ESCourseSerializer`, it didn't seem to be the case that `location` field is getting indexed for search right now (which makes some sense since the code responsible for syncing bootcamps was only recently merged in). Additionally, the data scope is very small. There are only 3 bootcamps in total, and only 1 of them has a location (bootcamp data is being sourced from here: https://github.com/mitodl/bootcamps.json/blob/master/bootcamps.json), so if it were the case that location was being indexed, this would only affect one bootcamp course. Location is null for all other courses. It's a field only for bootcamps that we recently added to the Course model because we squished bootcamps into Course. I'm happy to defer to you if you still feel strongly; we can totally do the model-change/migration in follow-up PR. Just let me know.

Reply:
@rhysyngsun can you confirm?

Reply:
Looks like that's correct